targets/TARGET_STM/TARGET_STM32WL/TARGET_STM32WLE5xC/TARGET_LORA_E5/PeripheralNames.h

```diff
@@ -0,0 +1,66 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_PERIPHERALNAMES_H
+#define MBED_PERIPHERALNAMES_H
+
```

Comment:
As you want, but I think we can move this file at TARGET_STM32WL level to be common for all ?

Reply:
absolutely, once gain, need knowledge of CMake, will try.

Reply:
are you ok to move 3 files `PeripheralNames.c` `PeripheralPins.h` `system_clock.c` to TARGET_STM32WL and keep only `PinNames.h` in target folder ?

Reply:
- PeripheralPins.c : no, specific for each target (you can manually disable or not a pin, you can add comment, etc...)
- PeripheralNames.h : yes
- system_clock.c: why not, as SetSysClock is weak

Reply:
ok done